src/app/runtime/wac/components/install.component.ts

```diff
@@ -10,20 +10,23 @@ const windowsPathValidationRegex = new RegExp('^(?:[a-z]:|\\\\\\\\[a-z0-9_.$●-
 @Component({
     template: `
     <div class="center">

```

Comment:
I think you should add sme-focus-zone to make the navigation with arrow keys work with the controls in this page.

Reply:
done